### PR TITLE
Dependency fixes and Development Update

### PR DESCRIPTION
**Pull request by qduk**
This PR pins the cloudvision dependency version as well as bringing the docker-compose method of development up to date with cookiecutter.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,14 +21,16 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-diffsync = "^1.3.0"
-cloudvision = "^1.0.0"
+python = "^3.7"
+nautobot = "^1.2.0"
 nautobot-ssot = "^1.0.1"
+Jinja2 = "<3"
```
Comment (glennmatthews) [inline pyproject.toml:27]: This will prevent usage with Nautobot 1.3.0 and later which require Jinja2 3.x.

**Review by glennmatthews — CHANGES_REQUESTED** [review]

**Review by jvanderaa — APPROVED** [review]

File: poetry.lock [hunk 2]
```diff
@@ -722,31 +729,16 @@ validation = ["swagger-spec-validator (>=2.1.0)"]
...
-importlib-metadata = {version = "*", markers = "python_version < \"3.8\""}
-mccabe = ">=0.6.0,<0.7.0"
-pycodestyle = ">=2.7.0,<2.8.0"
-pyflakes = ">=2.3.0,<2.4.0"
-
-[[package]]
-name = "flake8"
-version = "4.0.1"
-description = "the modular source code checker: pep8 pyflakes and co"
+version = "2.3.0"
```
[the remark this answers is not in the document]
Comment (qduk) [reply poetry.lock]: I recreated the `poetry.lock` file with the new flake version. Thanks Glenn!

File: pyproject.toml (hunk 1 above)
Comment (qduk) [reply pyproject.toml]: Updated

File: development/nautobot_config.py [hunk 3]
```diff
@@ -258,79 +78,5 @@ def is_truthy(arg):
         "from_cloudvision_default_device_status": "Active",
         "from_cloudvision_default_device_status_color": "ff0000",
         "apply_import_tag": True,
-        "cvaas_token": "",
```
Comment (glennmatthews) [inline development/nautobot_config.py:261]: Is this no longer needed for the plugin?

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**